BraveRewards/BraveRewards.framework/Headers/ledger.mojom.objc.h

```diff
@@ -158,7 +158,7 @@ typedef NS_ENUM(NSInteger, BATPromotionStatus) {
 
 typedef NS_ENUM(NSInteger, BATPlatform) {
   BATPlatformDesktop = 0,
-  BATPlatformAndroid = 1,
+  BATPlatformAndroidR = 1,
```

Comment:
Huh? is this intentional? "Android R"?

Reply:
Seems like it 😆

https://github.com/brave/brave-core/blob/00c5caf684aec24668e4e54c68974ea763c5eeb1/vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom#L358